### DISCUSSION TITLE
Support `endorse` package

### DISCUSSION THREAD

**Discussion by cmellman** [vincentarelbundock/marginaleffects#1143] [Ideas]
Hello! Since endorsement experiments are very useful for gauging people's opinions on contentious issues whilst removing social desirability bias, I think that it would be nice to add the option for using this package with the 'endorse' package in R. This is especially important since converting the endorsement model to another type of model could distort the results.

**Comment by vincentarelbundock** [comment]
Thanks for the suggestion. I opened an issue here, where I note some difficulties: https://github.com/vincentarelbundock/marginaleffects/issues/1144

My takeaway is that, unfortunately, the `predict()` function in this package may not be "complete" enough to be supported.
